src/adapter/etl-adapter-google-sheet/src/Flow/ETL/Adapter/GoogleSheet/SheetRange.php

```diff
@@ -34,7 +34,7 @@ public function nextRows(int $count) : self
 
         return new self(
             $this->columnRange,
-            $this->endRow + 1,
+            $count,
             $this->endRow + $count,
         );
```

Comment:
I left this method for now but IMO proper implementation is now in the extractor.